### PR TITLE
updated README and spec.json

### PR DESCRIPTION
**Pull request by andorsk**
updated README and specs.json file. Now resolvable url should be available for companion guide.

### CODE REVIEW

**Comment by andorsk** [comment]
image

File: docs/companion_guide.md [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
+**Editors:**
+~ [Daniel Buchner](https://www.linkedin.com/in/dbuchner/) (Block)
+
+**Contributors:**
+~ [Moises Jaramillo](https://www.linkedin.com/in/moisesjaramillo/)
+
+**Participate:**
+~ [GitHub repo](https://github.com/decentralized-identity/decentralized-web-node)
+~ [File a bug](https://github.com/decentralized-identity/decentralized-web-node/issues)
+~ [Commit history](https://github.com/decentralized-identity/decentralized-web-node/commits/main/docs/companion_guide.md)
```
Comment (TallTed) [inline docs/companion_guide.md:25]:
```suggestion
**Chairs**
- [Andor Kesselman](https://www.linkedin.com/in/andorsk/)
- [Liran Cohen](https://www.linkedin.com/in/itsliran/)

**Editors:**
- [Daniel Buchner](https://www.linkedin.com/in/dbuchner/) (Block)

**Contributors:**
- [Moises Jaramillo](https://www.linkedin.com/in/moisesjaramillo/)

**Participate:**
- [GitHub repo](https://github.com/decentralized-identity/decentralized-web-node)
- [File a bug](https://github.com/decentralized-identity/decentralized-web-node/issues)
- [Commit history](https://github.com/decentralized-identity/decentralized-web-node/commits/main/docs/companion_guide.md)
```

**Comment by andorsk** [comment]
@TallTed, while we haven't had the chance to meet in person, I've noticed your ongoing efforts in refining the specifications by addressing grammar and other aspects. Would you be interested in being added to the list of contributors? We'd welcome you on the list if you're open to being included.

**Comment by andorsk** [comment]
open for review

**Comment by TallTed** [comment]
> [@andorsk] Would you be interested in being added to the list of contributors? We'd welcome you on the list if you're open to being included.

Sure. You can add me anywhere that's a recognition of work I've already done.

**Review by LiranCohen — APPROVED** [review]

**Pull request merged**